modules/ethereum/src/lib.rs

```diff
@@ -44,6 +44,8 @@ mod benchmarking;
 
 #[cfg(test)]
 mod mock;
+#[cfg(test)]
+mod mock_chain_time;
```

Comment:
I think I'd rather make `mock.rs` file a module and have `mock::chain_time` there to prevent too many mocks from polluting `src` directory, wdyt?

Reply:
Would it be enough just to add `ConstChainTime` to existing `mock.rs` file?

Reply:
Started with that. Just adding a `TestConstantChainTimeRuntime` to `mock.rs`. Ended up with some duplicate items. Since I don't like having multiple `mock` files myself, I will give it another try.

Reply:
Why would you need new test runtime (`TestConstantChainTimeRuntime`) at all? :) Could you just add `type ChainTime = ConstChainTime;` to the existing `TestRuntime`? :)

Reply:
My goal was, to first of all, not break any existing tests. I don't know, whether the constant time chosen will interfere with any assumptions of the existing test. Hence, the new runtime. Can give it a try with just one runtime and see how it goes.

Reply:
I think most of tests will be fine. The rest needs to be adopted. I think it is better to have single test runtime - having separate runtime for every check is too much imo :)

Reply:
additional runtime removed.